mne/event.py

```diff
@@ -187,8 +188,10 @@ def _read_events_fif(fid, tree):
     event_list = event_list.reshape(len(event_list) // 3, 3)
     return event_list, mappings
 
-
-def read_events(filename, include=None, exclude=None, mask=0):
+@deprecated("The default setting for the argument 'mask_type' will change "
+            "from None to 'and' in v0.14.")
```

Comment:
Count is a lazy way of checking, really it should be something more like `assert_true(any('whatever' in str(ww.message) for ww in w))`, but you can see how much more annoying that is to get right than `assert_equal(len(w), 1)`...

You could fix the counts, but really it would be better to explicitly set the `mask_type='whatever'` in the places it will now fail. You should also really do this *anywhere a warning will currently be raised* because we don't want any un-caught warnings in our tests. But we're lax about checking for and enforcing that guideline, so you don't have to do it.